docs/react-native-install.md

```diff
@@ -152,7 +152,16 @@ Note 2: May need to Install `Node.js` if your `npm` command isn't working which
 
 ![goodYarnInstall](images/react_native/yarnInstallGood.png)
 
-## How to Install Node.js on Windows
+# Time to Run the Demo on Android
```

Comment:
there should be only 1 # (H1) per document.

Reply:
I know this comment is aimed at the section Erik is currently working on, but thanks for the reminder. I have markdownlint installed, looking at the information I'm pulling together for the macOS installation, it shows "First line in a file should be a top-level heading"

https://github.com/DavidAnson/markdownlint/blob/v0.27.0/doc/md041.md